packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx

```diff
@@ -198,6 +198,11 @@ export const menuTabs: ForumOptions<Array<MenuTab>> = {
       title: 'LW the Album',
       link: '/posts/YMo5PuXnZDwRjhHhE/the-story-of-i-have-been-a-good-bing',
       subItem: true
+    }, {
+      id: 'leaderboard',
+      title: 'Leaderboard',
+      link: '/leaderboard',
+      subItem: true
```

Comment:
Could you make this a LW only route? We will likely adopt this, but some of the text will need updating for us